src/internal/transforms/sqlingest.go

```diff
@@ -96,10 +100,48 @@ func makeWriterFactory(formatName string) (writerFactory, error) {
 		}
 	case "csv":
 		factory = func(w io.Writer, fieldNames []string) sdata.TupleWriter {
-			return sdata.NewCSVWriter(w, fieldNames)
+			return sdata.NewCSVWriter(w, nil)
 		}
 	default:
 		return nil, errors.Errorf("unrecognized format %v", formatName)
 	}
 	return factory, nil
 }
+
+type SQLQueryGenerationParams struct {
+	Logger              *logrus.Logger
+	InputDir, OutputDir string
+
+	Query string
+}
+
+// SQLQueryGeneration generates queries with a timestamp in the comments
+func SQLQueryGeneration(ctx context.Context, params SQLQueryGenerationParams) error {
+	timestamp, err := readCronTimestamp(params.Logger, params.InputDir)
+	if err != nil {
+		return err
+	}
+	timestampComment := fmt.Sprintf("-- %d\n", timestamp)
+	contents := timestampComment + params.Query
+	outputPath := filepath.Join(params.OutputDir, "0000")
```

Comment:
I'm missing something here. why "pfs/out/0000" ?

Reply:
`0000` is just the path that the query generation writes to by default.  I don't want users to get accustomed to something like RESULTS.json.  They should be writing code to consume an unknown number of files from the output, which will be based on how many separate queries they asked for in the input.  I'm open to suggestions here, I was thinking you would just increment this number if we ever start generating more than one query.